### ISSUE TITLE
[Bug]: Torchlight.dev components are not detected by plugin

### ISSUE THREAD

**Issue by miomech** [laravel-idea/plugin#665]
### Bug description

After installing torchlight.dev package the plugin constantly reports 

**Unknown Blade component**

example snippet:
```php
<x-torchlight-code language='c'>
    #include <stdio.h>
    int main() {
       // printf() displays the string inside quotation
       printf("Hello, World!"); // [tl! focus]
       return 0;
    }
</x-torchlight-code>
```

### Plugin version

6.0.0.222

### Operating system

Linux

### Steps to reproduce

Install on laravel project, attempt to use torchlight snippet. Torchlight will execute just fine but the plugins shows errors

### Relevant log output

```shell
None
```

**Comment by adelf** [comment]
Implemented in the 6.1 release.

**Comment by miomech** [comment]
Thank you very much for the help!
